substrate/frame/recovery/Cargo.toml

```diff
@@ -17,41 +17,25 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { features = ["derive"], workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
```

Comment:
god bless you, I hate seeing unused deps in pallet and always happy to see a bunch being removed :)) I think no tool is smart enough to detect this programmatically, `cargo machete` is close and I sometimes use it. But it could have false positives.